backend/src/main/resources/schema.graphqls

```diff
@@ -116,7 +123,7 @@ type Organization {
   testingFacility: [Facility]
 }
 # When starting from the patient, take a more limited view
-type PatientTestResult {
+type PatientTestResult { # TODO this isn't used anywhere; should it be deleted?
```

Comment:
yes this should be deleted. We initially showed all test results a person had on the edit person form. Plan is to replace this use case with a person search in the test results

Reply:
Added a note about this general class of problem to #1069, though it may be moot in this case.